src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitSection.jsx

```diff
@@ -100,8 +100,8 @@ export default function VideoVisitLocation({ header, appointment, facility }) {
             <div className="vads-u-margin-top--2">
               <VAFacilityLocation
                 facility={facility}
-                facilityId={appointment.videoData.facilityId}
-                clinicFriendlyName={appointment.location?.displayName}
+                facilityId={appointment.location.stationId}
+                clinicFriendlyName={appointment.location.clinicName}
```

Comment:
I thought clinicName was clearer, since saying displayName could be interpreted as the display name for the other ids in the location object, like station id or the vista id